src/containers/Event/Document/Document.js

```diff
@@ -1,16 +1,121 @@
-import React from "react";
+import React, { useState, useContext, useEffect } from "react";
+import {
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  IconButton,
+  Tooltip,
+  Collapse,
+} from "@material-ui/core";
+
+import {
+  Delete as DeleteIcon,
+  AddCircle as AddCircleIcon,
+} from "@material-ui/icons";
+
+import AddDoc from "./AddDoc";
 import "./Document.css";
+import { store } from "../../../store.js";
 
 function Document(props) {
+  const { currentUser, docs, deleteDoc } = useContext(store);
+  const [eventDocs, setEventDocs] = useState(docs);
+  const [docId, setDocId] = useState("");
+  const [docSrc, setDocSrc] = useState("");
+  const [expanded, setExpanded] = React.useState(false);
+
+  useEffect(() => {
+    setEventDocs(docs);
+    setDocId("");
+    setDocSrc("");
```

Comment:
This will reset `docId` and `docSrc` at every update. I'm sure it's not what you want